src/Symfony/Component/Cache/Adapter/Client/ClientInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter\Client;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+interface ClientInterface
+{
+    public static function isSupported();
+
+    public static function getRequirements();
+
+    public static function create($servers = array(), array $options = array());
+
+    public function getClient();
```

Comment:
Absolutely true. If you ignore that you don't like the idea of providing a wrapper for configuring `\Memcached`, would you have suggestions as to an interface that *would* make sense?